optimum/exporters/openvino/model_configs.py

```diff
@@ -1570,3 +1576,183 @@ def patch_model_for_export(
+        task: str,
+        normalized_config: NormalizedVisionConfig,
+        batch_size: int = DEFAULT_DUMMY_SHAPES["batch_size"],
+        num_channels: int = DEFAULT_DUMMY_SHAPES["num_channels"],
+        width: int = DEFAULT_DUMMY_SHAPES["width"],
+        height: int = DEFAULT_DUMMY_SHAPES["height"],
+        **kwargs,
+    ):
+        super().__init__(task, normalized_config, batch_size, num_channels, width, height, **kwargs)
+        if getattr(normalized_config, "in_channels", None):
+            self.num_channels = normalized_config.in_channels // 4
+
+    def generate(self, input_name: str, framework: str = "pt", int_dtype: str = "int64", float_dtype: str = "fp32"):
+        if input_name in ["hidden_states", "sample"]:
+            shape = [self.batch_size, (self.height // 2) * (self.width // 2), self.num_channels * 4]
+            return self.random_float_tensor(shape, framework=framework, dtype=float_dtype)
+        if input_name == "img_ids":
+            return self.prepare_image_ids(framework, int_dtype, float_dtype)
+
+        return super().generate(input_name, framework, int_dtype, float_dtype)
+
+    def prepare_image_ids(self, framework: str = "pt", int_dtype: str = "int64", float_dtype: str = "fp32"):
+        img_ids_height = self.height // 2
+        img_ids_width = self.width // 2
+        if framework == "pt":
+            import torch
+
+            latent_image_ids = torch.zeros(img_ids_height, img_ids_width, 3)
+            latent_image_ids[..., 1] = latent_image_ids[..., 1] + torch.arange(img_ids_height)[:, None]
+            latent_image_ids[..., 2] = latent_image_ids[..., 2] + torch.arange(img_ids_width)[None, :]
+
+            latent_image_id_height, latent_image_id_width, latent_image_id_channels = latent_image_ids.shape
+
+            latent_image_ids = latent_image_ids[None, :].repeat(self.batch_size, 1, 1, 1)
+            latent_image_ids = latent_image_ids.reshape(
+                self.batch_size, latent_image_id_height * latent_image_id_width, latent_image_id_channels
+            )
+            latent_image_ids.to(DTYPE_MAPPER.pt(float_dtype))
+            return latent_image_ids
+        if framework == "np":
+            import numpy as np
+
+            latent_image_ids = np.zeros(img_ids_height, img_ids_width, 3)
+            latent_image_ids[..., 1] = latent_image_ids[..., 1] + np.arange(img_ids_height)[:, None]
+            latent_image_ids[..., 2] = latent_image_ids[..., 2] + np.arange(img_ids_width)[None, :]
+
+            latent_image_id_height, latent_image_id_width, latent_image_id_channels = latent_image_ids.shape
+
+            latent_image_ids = np.tile(latent_image_ids[None, :], (self.batch_size, 1, 1, 1))
+            latent_image_ids = latent_image_ids.reshape(
+                self.batch_size, latent_image_id_height * latent_image_id_width, latent_image_id_channels
+            )
+            latent_image_ids.astype(DTYPE_MAPPER.np[float_dtype])
+            return latent_image_ids
```

Comment:
why not generate the random float with the final shape directly (self.batch_size, latent_image_id_height * latent_image_id_width, 3)

Reply:
because this info used as indices for iteration inside rope tensor, random values may raise IndexError for this case, so it is better to have determenistic result for this case

Reply:
the same is true for position_ids, random values in a small range (0, 1) wouldn't be an issue.

Reply:
I'll try but not sure as desired type is float, but values should be int